Form/ChoiceList/PhpcrOdmQueryBuilderLoader.php

```diff
@@ -84,6 +93,7 @@ public function getEntitiesByIds($identifier, array $values)
         }
```

Comment:
i am a bit unsure as to when this is executed, and if it is supposed to still respect other criteria if the querybuilder has them. like filtering for published only - otherwise i might be able to trick an unpublished document into the selection list even though the query would not find it. that is the only scenario i can imagine where there could be a problem with not using the QB.

in general however, i would expect the QB to handle queries for the id field correctly.